sessiontxn/staleread/processor.go

```diff
@@ -59,7 +60,8 @@ type baseProcessor struct {
 	is          infoschema.InfoSchema
 }
 
-func (p *baseProcessor) init(sctx sessionctx.Context) {
+func (p *baseProcessor) init(ctx context.Context, sctx sessionctx.Context) {
+	p.ctx = ctx
```

Comment:
If processor has the `context.Context`, can we replace `context.TODO()` with `p.ctx` in this file?

Reply:
Sure! Replaced :handshake: